src/ck-perf/trace-projections.h

```diff
@@ -48,27 +51,25 @@ class LogEntry {
     int msglen;
     int nestedID; // Nested thread ID, e.g. virtual AMPI rank number
     CmiObjId   id;
-    int numpes;
-    int *pes;
+    int numpes = 0;
```

Comment:
Do we need to store numpes explicitly here rather than just using pes.size()?

Reply:
Fixed